src/main/java/org/jboss/ejb/client/EJBClientContext.java

```diff
@@ -475,6 +487,20 @@ public void run() {
         }
     }
 
+    public void unregisterNodeEJBReceivers(String nodeName) {
```

Comment:
This is another API change that we can't include.  Essentially we cannot be adding public methods at this point, without a very good justification.